pkg/ebpf/bpf_client.go

```diff
@@ -749,7 +750,10 @@ func (l *bpfClient) computeMapEntriesFromEndpointRules(firewallRules []EbpfFirew
 	_, mapKey, _ := net.ParseCIDR(l.nodeIP + l.hostMask)
 	key := utils.ComputeTrieKey(*mapKey, l.enableIPv6)
 	value := utils.ComputeTrieValue([]v1alpha1.Port{}, l.logger, true, false)
-	mapEntries[string(key)] = uintptr(unsafe.Pointer(&value[0]))
+	firewallMap[string(key)] = value
+
+	//Sort the rules
+	sortFirewallRulesByPrefixLength(firewallRules, l.hostMask)
```

Comment:
We've an extra call for this function right below. Let's remove it.

Reply:
Thanks to git merge :D :D